lib/infer/return.js

```diff
@@ -17,7 +17,14 @@ module.exports = function () {
     if (comment.returns) {
       return comment;
     }
-    var fn = finders.findTarget(comment.context.ast);
+    var fn = findTarget(comment.context.ast);
+
+    // In case of `/** */ var x = function () {}` findTarget returns
+    // the declarator.
+    if (t.isVariableDeclarator(fn)) {
```

Comment:
I'm not sure, but do you want to update the top-level comment: `Infers returns tags by using Flow return type annotations` I'm not sure if this comment still applies int he same way.